### ISSUE TITLE
Ellipsis argument type is unknown for Python versions < 3.10

### ISSUE THREAD

**Issue by BeefaloKing** [microsoft/pyright#7531]
**Describe the bug**
When passing Ellipsis `...` as an argument Pylance/Pyright reports `Pylance(reportUnknownArgumentType)`, where mypy reports no issues (full output below).

Pylance/Pyright provided correct output in a different environment with python 3.11, I'm assuming the issue is in versions < 3.10.

Fwiw, I don't think the bug is specifically related to my use of a TypeVar, but that was the example I could come up with that didn't have other type errors due to the lack of access to `types.EllipsisType`.

**Code or Screenshots**
```python
from typing import TypeVar

_T = TypeVar('_T')
def identity(_v: _T, /) -> _T:
    return _v

if __name__ == '__main__':
    foo = identity(...)
    reveal_type(foo)
```

mypy reports:
```
test.py:9 Revealed type is "builtins.ellipsis"
Success: no issues found in 1 source file
```

Pylance/Pyright reports:
```
Type of "foo" is unknown Pylance(reportUnknownVariableType) [Ln 9, Col 5]
Argument type is unknown Pylance(reportUnknownArgumentType) [Ln 9, Col 20]
```

In my 3.11 environment, Pylance/Pyright does not experience the bug, and mypy reports:
```
test.py:9: note: Revealed type is "types.EllipsisType"
Success: no issues found in 1 source file
```

My only guess is this might have to do something with the fact we see `builtins.ellipsis` instead of `types.EllipsisType` for earlier versions.

**VS Code extension or command-line**
Running Pylance through the vscode extension. Reports as version `v2024.3.1`.

**Comment by erictraut** [comment]
I'm not able to repro the problem as reported. I tried installing pylance v2024.3.1, and it reports that `foo` is of type `EllipsisType` as expected. Here's the same code in [pyright playground](https://pyright-play.net/?strict=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoAqiApgGoCGIAsAFC0D6BUAvISRSABQDkj3AlLQAmxYJhEoYSeJ3oA3AFxRGAGigB6flAC0APmUEFtKCaghiMAK4gUyubVpIx9einIRiLlq170I5VBduIxpTKGAwMBZxYklpOE4AOmTBUNNzOWJyABt6eARiTgiwVKA) which reports the same.

Please double check that you're able to repro the reported behavior with a clean project and with the latest version of pylance.

**Comment by BeefaloKing** [comment]
Thanks for showing me the pyright playground, that makes it much easier to test version differences. Following your link, the issue reproduces there when Python Version is set to either `3.8` or `3.9`.

**Comment by erictraut** [comment]
This is by design. Prior to Python 3.10, the `EllipsisType` was not exposed as a public type. It is therefore not reflected in the typeshed stubs for these older versions of the language.

It's very atypical to use `...` in a value expression — especially as an argument to a function call. You might want to avoid doing this.
